bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/target/env/Arbitraries.scala

```diff
@@ -159,7 +160,148 @@ trait Arbitraries extends edu.gemini.spModel.core.Arbitraries {
   implicit val arbSingleAsterism: Arbitrary[Asterism.Single] =
     Arbitrary(arbitrary[SPTarget].map(Asterism.Single(_)))
 
-  implicit val arbAsterism: Arbitrary[Asterism] =
-    Arbitrary(arbitrary[Asterism.Single]) // TODO:ASTERISM: add GHOST
 
+  // Seal off all the generators for the pieces of GhostAsterisms.
+  private object GhostGens {
+    import GhostAsterism._
+    import GhostAsterism.GhostStandardResTargets._
+
+    // This generates a list of n coordinates, wherein the last n-1 are within distance [lower, upper) from the first.
+    def genNCoordsWithinDistance(n: Int, lower: Double, upper: Double): Gen[List[Coordinates]] =
+      for {
+        c  <- arbitrary[Coordinates]
+        cs <- listOfN(n-1, genCoordsWithinDistance(c, lower, upper))
+      } yield c :: cs
+
+    // TODO:GHOST Change these to GHOST constants as we learn more, and move to Ghost, probably.
```

Comment:
I also added a `TODO:GHOST` to go with `TODO:ASTERISM` so that we can `grep` through the codebase to remind ourselves what still needs to be done when we obtain more information.